common/src/main/java/com/microsoft/identity/common/internal/broker/InstallCertActivityLauncher.java

```diff
@@ -83,7 +83,10 @@ public final class InstallCertActivityLauncher extends AppCompatActivity {
     protected void onCreate(final Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         if (savedInstanceState == null) {
-            mInstallCertificateIntent = getIntent().getExtras().getParcelable(INSTALL_CERT_INTENT);
+            Bundle extras = getIntent().getExtras();
+            if(extras != null) {
+                mInstallCertificateIntent = extras.getParcelable(INSTALL_CERT_INTENT);
```

Comment:
Same question here, in what scenarios InstallCertActivity is being launched without the INSTALL_CERT_INTENT, and should we throw an error back?

Reply:
There is not much information available except for the crash stack. The partner also has closed the investigation due to low hits.
Even though it is good to understand the reason, we should prevent crashes by adding this check for sure.
I do not think we can return error here, but logging may be helpful.

Reply:
If we don't return error here then what happens? What is the end user experience in this case?